tests/unit/helpers/test_search.py

```diff
@@ -4,14 +4,17 @@
 
 import pytest
 
+import globus_sdk
 from globus_sdk import SearchQuery, SearchQueryV1, utils
 from globus_sdk.exc.warnings import RemovedInV4Warning
 
 
-@pytest.mark.filterwarnings("ignore:'SearchQuery'*:DeprecationWarning")
 def test_init_legacy():
     """Creates SearchQuery and verifies results"""
-    query = SearchQuery()
+    with pytest.warns(
+        globus_sdk.exc.RemovedInV4Warning, match="'SearchQuery' is deprecated"
+    ):
+        query = SearchQuery()
```

Comment:
These imports, and the subsequent usage here, aren't looking okay to me.

`globus_sdk` doesn't need to be imported, and the `RemovedInV4Warning` doesn't need to be qualified.

```python
    with pytest.warns(RemovedInV4Warning, match="'SearchQuery' is deprecated"):
        query = SearchQuery()
```

Reply:
Oh, :facepalm: , I missed that it was already imported by name.

That said, we document it as `globus_sdk.RemovedInV4Warning`, so I think it's best if the testsuite matches that usage. (Which is also not what I wrote!)

I'll make a change right now which is fairly minimal. Although I generally prefer the verbose style of importing a package name (in our case, `globus_sdk`) and accessing all of the qualified names we need from that name, rewriting the whole test module to use that style isn't the small change I had in mind for this PR.